otter/test/models/test_cass_models.py

```diff
@@ -3353,7 +3312,28 @@ def test_list_states_deletes_resurrected_groups(self):
             mock.call(expectedCql, expectedData, ConsistencyLevel.QUORUM))
         self.assertEqual(r, [GroupState(tenant_id='123',
                                         group_id='group123',
-                                        group_name='test123',
+                                        group_name='test',
+                                        active={},
+                                        pending={},
+                                        group_touched='0001-01-01T00:00:00Z',
+                                        policy_touched={},
+                                        paused=False)])
+
+    def test_list_states_filters_deleting_groups(self):
+        """
+        Groups that are deleting based on "deleting" column to be true are not
+        returned
```

Comment:
Should we also assert that no call to delete the deleted group is made?  (It's only filtered, and not further deleted)